I replaced spell miss code.

fprg opened the pull request:
I replaced authenticateWithCompletionBlock method code.
autheticateWithCompletionBlock
↓
authenticateWithCompletionBlock

fprg committed:
I replaced spell miss code.

I replaced authenticateWithCompletionBlock method code.
autheticateWithCompletionBlock
↓
authenticateWithCompletionBlock

boxcla commented:
Hi @fprg, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

fprg committed:
I replaced spell miss code.

I replaced authenticateWithCompletionBlock method code.
BOXFoderItemsRequest *folderItemsRequest
↓
BOXFolderItemsRequest *folderItemsRequest

boxtcarpel commented:
+2 LGTM, we just need to rebase the branch since it seems to have gotten out of date.

Thanks @fprg for fixing these misspellings!

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/box/box-ios-sdk?pullRequest=241) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/box/box-ios-sdk?pullRequest=241) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/box/box-ios-sdk?pullRequest=241) it.</sub>
